R/client.R

```diff
@@ -123,7 +123,7 @@ isContentType <- function(x, contentType) {
   grepl(contentType, x, fixed = TRUE)
 }
 
-
+# These functions should move to client-shinyapps.R
```

Comment:
Just an observation, everything else is stashed in a `client` object, except this one and the function below it. Not touching the shinyapps code in this PR though.